libraries/joomla/session/storage.php

```
@@ -65,7 +65,7 @@ public static function getInstance($name = 'none', $options = array())
 				}
 				else
 				{
-					// No call to JError::raiseError here, as it tries to close the non-existing session
+					// Attempt to close the non-existing session
 					jexit('Unable to load session storage class: ' . $name);
```

Should we throw an exception here?